cpp/include/cudf/detail/datetime.hpp

```diff
@@ -26,7 +26,8 @@ namespace CUDF_EXPORT cudf {
 namespace datetime {
 namespace detail {
 /**
- * @copydoc cudf::extract_year(cudf::column_view const&, rmm::device_async_resource_ref)
+ * @copydoc cudf::extract_year(cudf::column_view const&, rmm::cuda_stream_view,
+ * rmm::device_async_resource_ref)
  *
  * @param stream CUDA stream used for device memory operations and kernel launches.
```

Comment:
Remove `@param stream` docstring from all other instances too, since it's already added to the copied doc.
```suggestion
```

Reply:
thanks!